operator/pkg/utils/utilities.go

```diff
@@ -6,13 +6,64 @@ package utils
 import (
 	"os"
 	"strings"
+	"reflect"
+	"math"
 )
 
 func IsPSPEnabled() bool {
 	value, exists := os.LookupEnv("ENABLE_VMWARE_PSP")
 	return exists && "true" == strings.TrimSpace(value)
 }
 
+func RangeInt(min, max, step int) []int {
+	size := int(math.Ceil(float64((max - min)) / float64(step)))
+	l := make([]int, size)
+	for i := 0; i < size; i++ {
+		l[i] = min + i * step
+	}
+	return l
+}
```

Comment:
this is one where there's been folks debating it in the golang community many times, and the idiom is just use a for loop inlined...
```
for i := min; i < max; i += step {
    doSomething(i)
}
```

Reply:
I actually need a slice of ints. See the code where I use this.